### PR TITLE
{lib,vis}[foss/2022a] Albumentations v1.3.0, imgaug v0.4.0 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by crubb**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/a/Albumentations/Albumentations-1.3.0-foss-2022a-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('PyYAML', '6.0'),
+    ('scikit-image', '0.19.3'),
+    ('scikit-learn', '1.1.2'),
+    ('OpenCV', '4.6.0', '-CUDA-%(cudaver)s' + '-contrib'),
```
Comment (smoors) [inline easybuild/easyconfigs/a/Albumentations/Albumentations-1.3.0-foss-2022a-CUDA-11.7.0.eb:19]:
```suggestion
    ('OpenCV', '4.6.0', versionsuffix + '-contrib'),
```

File: easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2022a-CUDA-11.7.0.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('Python', '3.10.4'),
+    ('Pillow', '9.1.1'),
+    ('matplotlib', '3.5.2'),
+    ('scikit-image', '0.19.3'),
+    ('OpenCV', '4.6.0', '-CUDA-%(cudaver)s' + '-contrib'),
```
Comment (smoors) [inline easybuild/easyconfigs/i/imgaug/imgaug-0.4.0-foss-2022a-CUDA-11.7.0.eb:20]:
```suggestion
    ('OpenCV', '4.6.0', versionsuffix + '-contrib'),
```

File: easybuild/easyconfigs/a/Albumentations/Albumentations-1.3.0-foss-2022a-CUDA-11.7.0.eb (hunk 1 above)
Comment (crubb) [reply easybuild/easyconfigs/a/Albumentations/Albumentations-1.3.0-foss-2022a-CUDA-11.7.0.eb]: Why did I do that? ;) Both fixed!

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by crubb** [comment]
Wasn't me ;)

 * **FAIL (build issue)** _OpenCV-4.6.0-foss-2022a-CUDA-11.7.0-contrib.eb_ (partial log available at https://gist.github.com/3f9e01f761e870e9638b5a3d241edcfd):

`== 2022-12-28 13:57:53,106 build_log.py:169 ERROR EasyBuild crashed with an error (at easybuild/easybuild-framework/easybuild/base/exceptions.py:124 in __init__): Don't know how to configure OpenCV in accordance with --optarch='{'Intel': 'march=core-avx2'}' (at easybuild/easybuild-easyblocks/easybuild/easyblocks/o/opencv.py:175 in configure_step)`

Thanks for testing!

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Comment by smoors** [comment]
> **FAIL (build issue)** _OpenCV-4.6.0-foss-2022a-CUDA-11.7.0-contrib.eb_ (partial log available at https://gist.github.com/3f9e01f761e870e9638b5a3d241edcfd):

OpenCV built in jsc-zen2 in PR https://github.com/easybuilders/easybuild-easyconfigs/pull/16646, should be running fine now.

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @crubb!

**Pull request merged**